packages/app/src/__tests__/utils.test.ts

```diff
@@ -73,3 +80,78 @@ describe('formatAttributeClause', () => {
     );
   });
 });
+
+describe('getMetricTableName', () => {
+  // Base source object with required properties
+  const createBaseSource = () => ({
+    from: {
+      tableName: 'default_table',
+      databaseName: 'test_db',
+    },
+    id: 'test-id',
+    name: 'test-source',
+    timestampValueExpression: 'timestamp',
+    connection: 'test-connection',
+    kind: 'logs' as const,
```

Comment:
nit: 'metric'?

Reply:
I meant to overwrite that oops. Fixing rq